browser/brave_profile_prefs.cc

```diff
@@ -277,7 +277,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
 
   // Not using chrome's web service for resolving navigation errors
   registry->SetDefaultPrefValue(embedder_support::kAlternateErrorPagesEnabled,
-                                base::Value(false));
+                                base::Value(true));
```

Comment:
I think this line change should have a sec review.

Reply:
@fmarier wdyt?

Reply:
@spylogsster do we actually need this `SetDefaultPrefValue` call anymore? I thought it's already the default behavior in Chromium.

Reply:
https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/net/profile_network_context_service.cc;l=403?q=kAlternateErrorPagesEnabled&ss=chromium&start=21 Looks like we can remove it if no other reasons.

Reply:
Separately from the discussion around enabling this feature flag though, we should have a sec review because this adds new network requests to the browser (namely requests to http://detectportal.brave-http-only.com).

@spylogsster Could you please file a sec review issue for it?

Reply:
reworked and created sec review issue, https://github.com/brave/security/issues/1217